### PR TITLE
Task 876 fix redirect after success login

### PR DESCRIPTION
**Pull request by oni-hash**
## Description

When user logins sucessfully, he's now redirected to homepage, not to profile-editing page

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| ** original screenshot ** | ** updated screenshot ** |

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [ ] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [ ] 🔗 Link pull request to issue

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```
Comment (maarekua) [inline .vscode/settings.json:2]: You should put the .vscode folder to ignore in the .git configurtaion.

File: src/pages/product-details/product-images/product-images.js [hunk 2]
```diff
@@ -77,7 +69,7 @@ const ProductImages = () => {
         onClick={() => openImage(i + 1)}
       />
     ));
-
+  console.log(`img${IMG_URL}${images.primary.large}`);
```
Comment (maarekua) [inline src/pages/product-details/product-images/product-images.js:72]: Please remove console.log.

File: src/pages/product-details/product-images/product-images.js [hunk 3]
```diff
@@ -94,8 +86,15 @@ const ProductImages = () => {
         rightArrowTitle={IMGS_VIEWER[language].next}
       />
       <div className={styles.images}>
-        {sideImages}
-        {primaryImage}
+        <div>{sideImages}</div>
+        <div className={styles.imagePreviewContainer}>
+          <Zoom
+            img={IMG_URL + images.primary.large}
+            zoomScale={3}
```
Comment (maarekua) [inline src/pages/product-details/product-images/product-images.js:93]: All configurations of zoom should be moved to constant.

File: .vscode/settings.json (hunk 1 above)
Comment (vasyaukr) [reply .vscode/settings.json:2]: or just remove this file. Its a problem of IDE

File: src/pages/product-details/product-images/product-images.styles.js [hunk 4]
```diff
@@ -14,32 +14,32 @@ export const useStyles = makeStyles((theme) => ({
...
+
     '@media (max-width: 1600px)': {
       width: 'auto',
       height: 'auto'
     }
   }),
+
   sideImage: {
+    width: '100%',
+
```
Comment (vasyaukr) [inline src/pages/product-details/product-images/product-images.styles.js:40]: u shouldn't use gaps between this simple styles

**Pull request closed without merging**